docs/docs/modeling.md

```diff
@@ -104,6 +104,35 @@ classDiagram
 
 ## Owner - オーナー
 
+### Owner
+
+The Owner contains ShareHouse.
+
+### Owner ID (Identifier)
+
+The Owner ID is a UUID.
```

Comment:
Is it supposed to be a UUID?

Reply:
I guess "unique identifier (UID) " is appropriate in this context.

Reply:
As we discussed, we decided to make use of UUID as a ID.